src/test/java/net/hydromatic/sqllogictest/ITestRunWithJars.java

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
no reason for this file to have a vmware copyright

Reply:
Yeap that's true but didn't find an easy way to make checkstyle accept two licenses so decided to ignore this for the moment.